libraries/AP_DroneCAN/AP_Canard_iface.cpp

```diff
@@ -176,6 +176,14 @@ void CanardInterface::processTx() {
         // scan through list of pending transfers
         while (true) {
             auto txf = &txq->frame;
+            if (raw_commands_only && ((txf->id >> 8) & UAVCAN_EQUIPMENT_ESC_RAWCOMMAND_ID) != UAVCAN_EQUIPMENT_ESC_RAWCOMMAND_ID) {
```

Comment:
better to check transfer priority? ESC_RAWCOMMAND is not the only one we need to do (eg. new Hobbywing RAW command)

Reply:
Just added